wiki/Project_Loved/en.md

```diff
@@ -0,0 +1,38 @@
+# Project Loved
+
+**Project Loved** is the voting platform that promotes beatmaps to the [Loved](/wiki/Beatmaps#loved) category. It is run primarily by the [Project Loved Team](/wiki/People/Project_Loved_Team).
+
+## Moving a beatmap to Loved
+
+Beatmaps are moved to Loved by a nomination and voting process.
+
+Members of the Project Loved Team called **captains** regularly nominate beatmapsets for a specific game mode. Almost always, these sets come from submissions from the osu! community. The current Loved submission forms and listings can be found via the following links:
+
+- [osu!/osu!taiko/osu!catch form](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)
+- [osu!mania form](https://docs.google.com/forms/d/e/1FAIpQLSeaGfoQNGMqw4qQcqRPItUZILh2fGwJR6ly6cZNY9OWPXkFhw/viewform)
+- [osu!/osu!taiko/osu!catch listing](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit)
+- [osu!mania listing](https://docs.google.com/spreadsheets/d/1sjkTwUSvQ5Me-6rK61rToTg2bU-yX9X29CXdzttvhtM/edit)
+
+*Note: If your access to Google Forms is restricted, you may also contact any of the [Project Loved Team members](/wiki/People/Project_Loved_Team#team-members) to have a map submitted on your behalf.*
+
+Nominated mapsets are then posted to the [Project Loved forum](https://osu.ppy.sh/community/forums/120), where community members may vote whether they want the mapset to be Loved or not. Maps reaching a specified threshold percentage of "Yes" votes at their polls' end time will be moved to the Loved category.
+
+### Eligibility
+
+There are some very minimal criteria that need to be met for beatmaps to be approved for Loved. Captains consider a beatmapset eligible for the Loved category if:
+
+- It is not in the [Qualified](/wiki/Beatmaps#qualified), [Ranked](/wiki/Beatmaps#ranked) or [Approved](/wiki/Beatmaps#approved) categories
```

Comment:
```suggestion
- It is not [qualified](/wiki/Beatmaps#qualified), [ranked](/wiki/Beatmaps#ranked) nor [approved](/wiki/Beatmaps#approved)
```

Reply:
I've been using "Loved category" throughout this article and that's how it's references everywhere else so I did the same for other categories

also "or" is correct

Reply:
marking as resolved unless if we want to change the "<Map> category" stuff elsewhere